src/frontend/src/session.rs

```diff
@@ -303,6 +305,9 @@ impl FrontendEnv {
 
         let system_params_manager =
             Arc::new(LocalSystemParamsManager::new(system_params_reader.clone()));
+        let session_params = Arc::new(RwLock::new(
```

Comment:
The recovery test panicked here:
```
2022-09-01T05:12:22.818087Z  INFO risingwave_simulation::cluster: kill meta-2
thread '<unnamed>' panicked at src/frontend/src/lib.rs:170:72:
called `Result::unwrap()` on an `Err` value: gRPC request failed: The service is currently unavailable: transport error
```
Since meta could be killed at this point, can we introduce a session params reader as well like system params reader?

Reply:
No, I can just use the MetaSnapshot